test/integration/integration_test.go

```diff
@@ -0,0 +1,64 @@
+// +build integration
+
+/*
+Copyright (C) 2017 Red Hat, Inc.
```

Comment:
I think we can remove this license. In Minishift we had some check which made us to use it.